glean_parser/markdown.py

```diff
@@ -168,13 +168,14 @@ def output_markdown(
             # the description
             if isinstance(obj, pings.Ping):
                 custom_pings_cache[obj.name] = obj
+                # Pings that have `send_if_empty` set to true,
+                # might not have any metrics.
                 if obj.send_if_empty:
                     metrics_by_pings[obj.name] = []
```

Comment:
I just realized that this might overwrite metrics that were added prior to it. I'll fix it and send a new commit.

Reply:
Cool, please flag me again once it's ready

Reply:
It's ready!